core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -2960,7 +2960,7 @@ private RecordCursorFactory generateSelectChoose(QueryModel model, SqlExecutionC
                 QueryColumn qc = columns.getQuick(i);
                 if (
                         !Chars.equals(metadata.getColumnName(i), qc.getAst().token) ||
```

Comment:
nit: again, unrelated with the PR, but we should use `equalsIgnoreCase` here:
```suggestion
                        !Chars.equalsIgnoreCase(metadata.getColumnName(i), qc.getAst().token) ||
```